src/app/modules/core-bcp/components/core-facility-info/core-facility-info.component.ts

```diff
@@ -34,4 +34,11 @@ export class CoreFacilityInfoComponent {
   registerOnTouched(fn: any): void {
```

Comment:
Please remove variables/functions that are not used.  There are functions/variables in this file that are related to custom form controls (registerOnTouch, registerOnChange)

Reply:
Removed.